packages/backend-api/install/composer.json.patch

```diff
@@ -0,0 +1,8 @@
+@@ -85,6 +85,7 @@
+         "symfony-cmf/routing": "^2.0.3",
+         "symfony-cmf/routing-bundle": "^2.0.3",
+         "tracy/tracy": "^2.4.13",
++        "trikoder/oauth2-bundle": "^1.1",
```

Comment:
Why do you need to patch composer.json? Can this dependency be in API package?

Reply:
Well... complicated question.

The backend API itself doesn't need the `oauth2` bundle. The `oauth2` bundle is needed only for securing the API, and that is a responsibility of the `project-base`.

Additionally we have to mention packages we use directly from `project-base` in the project-base `composer.json`, and because we register the bundle in `AppKernel`, we have to mention it.

Reply:
I understand responsibilities.
But look at it from a different perspective. Information that API should be secured is already present in the backend-api package (install patches). 
And if you need API package, you need to require it in composer.
For ease installation (less files to patch, no need to run extra composer update) I'm opened to have the dependency in backend-api package.

Reply:
Check this during BV
* the proposed way would result in shorter documentation and less patches
* the proposed way is not "correct" in terms of naming transitive dependencies when we depend on them directly

Reply:
`trikoder/oauth2-bundle` is only important for the integration part in `shopsys/project-base` (after the installation). I'm fine with complicating the installation / uninstallation in project repository this way in exchange for the benefit of having requirements where they are supposed to be.